public/wp-content/themes/glific/page-pricing.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+/**
+ * Pricing page for the theme glific
+ *
+ * @package glific
+ */
+get_header();
+$cover_content = get_field('cover_content');
+?>
+<div class="page-pricing pb-26 pt-14 pt-md-18 bg-theme-white-smoke">
+	<h1 class="text-center fz-28 leading-40 fz-xl-36 leading-40 font-heebo-bold text-theme-primary"><?php echo get_the_title(); ?></h1>
+	<?php $cover_content = get_field('cover_content');
+	if(!empty($cover_content)) : ?>
+	<div class="font-heebo-regular fz-18 leading-28 w-328 w-md-641 w-xl-830 mx-auto mt-10 mt-md-14 mt-xl-18">
+		<?php echo $cover_content ; ?>
+	</div>
+	<?php endif;
+	$details = get_field('details');
+	if(!empty($details)) : ?>
+	<div class="bg-white w-328 w-md-689 w-xl-900 mx-auto p-6 p-md-8 rounded-30 box-shadow-dark-20 mt-26">
+	<?php foreach ($details  as $key => $detail) : ?>
+		<div class="d-flex flex-column <?php echo $key == count( $details ) - 1 ? '' : 'mb-10 mb-xl-14' ; ?>">
```

Comment:
Are we checking for the last element?
Check it our in that case https://www.php.net/manual/en/function.array-key-last.php
We can use `array_key_last()` in PHP 7+

Reply:
@Satendra-SR It requires  PHP minimum version of 7.3. I am not sure that, are our servers up to date with PHP 7.3 or not. thoughts ?

Reply:
Ah! Interesting. We are not sure in which version of PHp it will be deployed. You may want to try this. https://stackoverflow.com/questions/2348205/how-to-get-last-key-in-an-array

Reply:
https://www.php.net/manual/en/function.array-key-last.php#123016